common/worldFactory.h

```diff
@@ -1,18 +0,0 @@
-#ifndef WorldFactory_h
```

Comment:
Curious question, why did you delete this file? It an a few others were pretty important.